<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuData_Tambang">
    <property name="title">
     <string>Data Tambang</string>
    </property>
    <addaction name="actionDataIup"/>
   </widget>
   <widget class="QMenu" name="menuMasterData">
    <property name="title">
     <string>Master Data</string>
    </property>
    <addaction name="actionDataDesa"/>
    <addaction name="actionData_Sungai"/>
    <addaction name="actionDataPemukiman"/>
   </widget>
   <widget class="QMenu" name="menuKeluar">
    <property name="title">
     <string>Keluar</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuData_Tambang"/>
   <addaction name="menuMasterData"/>
   <addaction name="menuKeluar"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionDataIup">
   <property name="text">
    <string>Data IUP</string>
   </property>
  </action>
  <action name="actionDataDesa">
   <property name="text">
    <string>Data Desa</string>
   </property>
  </action>
  <action name="actionData_Sungai">
   <property name="text">
    <string>Data Sungai</string>
   </property>
  </action>
  <action name="actionDataPemukiman">
   <property name="text">
    <string>Data Pemukiman</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
